ReactAndroid/src/main/res/devsupport/values/strings.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <string name="catalyst_reloadjs" project="catalyst" translatable="false">Reload JS</string>
-  <string name="catalyst_debugjs" project="catalyst" translatable="false">Debug in Chrome</string>
-  <string name="catalyst_debugjs_off" project="catalyst" translatable="false">Stop Chrome Debugging</string>
+  <string name="catalyst_debugjs" project="catalyst" translatable="false">Debug JS Remotely</string>
```

Comment:
@ide said: "Debug JavaScript Remotely (or Debug JS Remotely if that string fits in the UI more nicely) sounds good to me. The "remotely" part is fairly important to make clear."
@mkonicek Could you please agree with @ide on what string to use? I'm happy to change it to whatever you guys agree is clearer/better